plugins/spf.js

```diff
@@ -274,15 +273,6 @@ exports.save_to_header = function (connection, spf, result, mfrom, host, id, ip)
     if (!connection) return;
     if (!connection.transaction) return;
     connection.transaction.add_leading_header('Received-SPF',
-        spf.result(result) +
-        ' (' + plugin.config.get('me') + ': domain of ' + host +
-        ((result === spf.SPF_PASS) ? ' designates ' : ' does not designate ') +
-        connection.remote.ip + ' as permitted sender) ' + [
-            'receiver=' + plugin.config.get('me'),
-            'identity=' + id,
-            'client-ip=' + (ip ? ip : connection.remote.ip),
-            'helo=' + connection.hello.host,
-            'envelope-from=<' + mfrom + '>'
-        ].join('; ')
+        `${spf.result(result)} (${plugin.config.get('me')}: domain of ${host}${result === spf.SPF_PASS ? ' designates ' : ' does not designate '}${connection.remote.ip} as permitted sender) ${[`receiver=${plugin.config.get('me')}`, `identity={id}`, `client-ip=${ip ? ip : connection.remote.ip}`, `helo=${connection.hello.host}`, `envelope-from=<${mfrom}>`].join('; ')}`
```

Comment:
With a string literal, we can get rid of the join('; ') here too?

Reply:
Sure, we can. I'll make the changes and commit.

Reply:
In fact, I missed `$` too here in  `identity=${id}`.